winpr/libwinpr/sspi/test/TestAcquireCredentialsHandle.c

```diff
@@ -22,10 +22,18 @@ int TestAcquireCredentialsHandle(int argc, char* argv[])
 	table = InitSecurityInterface();
 
 	identity.User = (UINT16*) _strdup(test_User);
-	identity.UserLength = sizeof(test_User);
 	identity.Domain = (UINT16*) _strdup(test_Domain);
-	identity.DomainLength = sizeof(test_Domain);
 	identity.Password = (UINT16*) _strdup(test_Password);
+	if (!identity.User || !identity.Domain || !identity.Password)
+	{
+		free(identity.User);
+		free(identity.Domain);
+		free(identity.Password);
+		fprintf(stderr, "Memory allocation failed\n");
+		return -1;
+	}
+	identity.UserLength = sizeof(test_User);
+	identity.DomainLength = sizeof(test_Domain);
```

Comment:
Not your code but these two sizeof() calls are wrong since it gives the pointer size !!